ShellPkg/Library/UefiShellDebug1CommandsLib/SmbiosView/PrintInfo.c

```diff
@@ -382,6 +382,24 @@ SmbiosPrintStructure (
         PRINT_PENDING_STRING (Struct, Type2, LocationInChassis);
         PRINT_STRUCT_VALUE_H (Struct, Type2, ChassisHandle);
         DisplayBaseBoardBoardType (Struct->Type2->BoardType, Option);
+        ShellPrintHiiEx (
+          -1,
+          -1,
+          NULL,
+          STRING_TOKEN (STR_SMBIOSVIEW_PRINTINFO_NUM_CONTAINED_OBJECTS_HANDLES),
+          gShellDebug1HiiHandle,
+          Struct->Type2->NumberOfContainedObjectHandles
+          );
+        for (Index = 0; Index < Struct->Type2->NumberOfContainedObjectHandles; Index++) {
+          ShellPrintHiiEx (

```

Comment:
Just one NIT: would it be possible to use `ShellPrintHiiDefaultEx()` instead ? Same comment for the other patch